agent/task.go

```diff
@@ -249,6 +249,7 @@ func (a *Agent) makeTaskConfig(ctx context.Context, tc *taskContext) (*model.Tas
 	}
 	grip.Info("Fetching distro configuration.")
 	confDistro, err := a.comm.GetDistro(ctx, tc.task)
+	grip.Infof("task.go, 252, Err: %s ", err)
```

Comment:
I think you probably don't want to include this in the final output. 

you can also do message.WrapStack(0, <thing>) to get a message with a stack frame automatically added.

Reply:
Thank you! message.WrapStack would have been super helpful. I'll keep it in mind for next time. 
I'm going to clean up all my logging info. I pushed it to this branch so that Jonathan can push it to staging for me. Apologies for the mess.